AppServer/google/appengine/api/taskqueue/taskqueue_distributed.py

```diff
@@ -102,6 +106,7 @@ def __init__(self, app_id, host, service_name='taskqueue'):
         service_name, max_request_size=MAX_REQUEST_SIZE)
     self.__app_id = app_id
     self.__nginx_host = host
+    self.__tq_locations = self._GetTQLocations()
```

Comment:
This has the downside of requiring a process restart if the locations file changes, but I think that's acceptable for now.

Reply:
Good point.
If nobody minds, I'd prefer to keep it as it is here.
Later we'd have to implement zookeeper watches for services locations in AppServer.